ArduCopter/mode_autorotate.cpp

```diff
@@ -259,7 +259,7 @@ void ModeAutorotate::run()
         // Update controllers
         pos_control->update_z_controller();
 
-        if ((now - _bail_time_start)/1000.0f >= _bail_time) {
+        if ((now - _bail_time_start_ms)/1000.0f >= _bail_time) {
```

Comment:
It's all getting better, but this could have been done with a multiply on the RHS rather than a divide on the LHS.  Similarly elsewhere.  This is a rapidly-called function so it is worth considering.

Reply:
Good point!  I have taken this under advisement and will apply your recommendation in a future autorotation mode PR of mine.  As this is still only available for SITL it shouldn't be an issue as is.